src/parse_sync.cc

```diff
@@ -52,9 +52,9 @@ NAN_METHOD(protagonist::ParseSync) {
     drafter::ParseBlueprint(*sourceData, options, parseResult);
 
     if (parseResult.report.error.code != snowcrash::Error::OK) {
-        NanThrowError(SourceAnnotation::WrapSourceAnnotation(parseResult.report.error));
-        NanReturnUndefined();
+        Nan::ThrowError(SourceAnnotation::WrapSourceAnnotation(parseResult.report.error).As<v8::Value>());
```

Comment:
As far as I can see, `SourceAnnotation::WrapSourceAnnotation` is used to create a new object type for warnings and errors. It might be too much to have it return a `Nan::Error` since it could be “just” a warning? But maybe it could return a `v8::Local<v8::Value>` instead of `Local<Object>`?